src/sentry/static/sentry/app/views/performance/transactionSummary/userStats.tsx

```diff
@@ -40,17 +48,39 @@ class UserStats extends React.Component<Props> {
     return eventView;
   }
 
-  renderContents(stats: Results) {
+  renderContents(stats: Results, row?) {
+    let userMisery = <StatNumber>{'\u2014'}</StatNumber>;
+
+    if (stats) {
+      const miserableUsers = Number(row[`user_misery_${userMiseryLimit}`]);
+      const totalUsers = Number(row.count_unique);
+      if (!isNaN(miserableUsers) && !isNaN(totalUsers)) {
+        userMisery = (
+          <UserMisery
+            bars={40}
+            barHeight={30}
+            miseryLimit={userMiseryLimit}
+            totalUsers={totalUsers}
+            miserableUsers={miserableUsers}
+          />
+        );
+      }
+    }
+
     return (
       <Container>
         <div>
-          <StatTitle>{t('Apdex Score')}</StatTitle>
+          <SectionHeading>{t('Apdex Score')}</SectionHeading>
           <StatNumber>{!stats ? '\u2014' : stats['apdex()']}</StatNumber>
         </div>
         <div>
-          <StatTitle>{t('User Misery')}</StatTitle>
-          <StatNumber>{!stats ? '\u2014' : stats['user_misery(300)']}</StatNumber>
+          <SectionHeading>{t('Baseline Duration')}</SectionHeading>
```

Comment:
I wouldn't include Baseline Duration until we have data to put here (which could be a while as we haven't really defined what baseline is going to be yet).

Reply:
Hid it!